index.html

```diff
@@ -494,6 +494,10 @@ <h2>Glossary</h2>
 
 <p><dfn  class="lint-ignore export">User-supplied value</dfn>. Unreserved [=syntactic content=] in a [=vocabulary=] that is assigned by users, as distinct from reserved keywords in a given format or protocol. Users generally expect that their user-supplied values can be words or phrases in their preferred [=natural language=]. This is why [[CHARMOD]] recommends that "Specifications <em class="rfc2119">SHOULD NOT</em> arbitrarily exclude code points from the full range of Unicode code points from <code>U+0000</code> to <code>U+10FFFF</code> inclusive." <!-- [[CHARMOD-NORM]] gives <a href="https://w3c.github.io/charmod-norm/#dfn-user-supplied-value">some examples</a>.  ADD THIS BACK ONCE CHARMOD-NORM IS UPDATED WITH THESE DEFINITIONS --></p>
 
+<p><dfn class="lint-ignore">UTF-8</dfn>. A variable-length [=character encoding=] using fewer bytes to encode [=code points=] with lower numerical values. For details of its definition, see <a href="https://www.unicode.org/versions/latest/ch02.pdf#G13708">Section 2.5, Encoding Forms</a> and <a href="https://www.unicode.org/versions/latest/ch03.pdf#G7404">Section 3.9, Unicode Encoding Forms</a> in The Unicode Standard.</p>
```

Comment:
Where is this definition taken from? Unicode?

Reply:
Note that the Unicode glossary has (or has recently developed?) a definition

Reply:
Should I quote the definition at https://unicode.org/glossary/#UTF_8 ?